proto/xudrpc.proto

```diff
@@ -139,7 +139,8 @@ message GetInfoResponse {
   int32 num_pairs = 2 [json_name = "num_pairs"];
   string version = 3 [json_name = "version"];
   OrdersCount orders = 4 [json_name = "orders"];
-  LndInfo lnd = 5 [json_name = "lnd"];
+  LndInfo lndbtc = 5 [json_name = "lndbtc"];
+  LndInfo lndltc = 7 [json_name = "lndltc"];
```

Comment:
We're early enough in development that I don't think there are backwards compatibility concerns with proto field numbering, I'd prefer if this was named 6 (and raiden below 7) to keep them ordered.

Reply:
not sure why you think we should keep the sequence here (what value does it give beside being unique?)  but  I did it.

done.